src/Apps/Fairs/Routes/FairsIndex.tsx

```diff
@@ -260,16 +259,6 @@ export const FairsIndex: React.FC<React.PropsWithChildren<FairsIndexProps>> = ({
                     </Column>
                   )
                 })}
-
-                {
-                  // Avoids orphaned CTA
-                  currentFairs[currentFairs.length - 1].bannerSize !==
-                    "x-large" && (
-                    <Column span={6}>
-                      <FairsPhonePromo pt={2} />
-                    </Column>
-                  )
-                }
```

Comment:
Question about this @damassi : I noticed that this page was displaying additional fairs in production that I don't see locally, so this <FairsPhonePromo /> component was not actually rendering for me when I removed it (the last fair.bannerSize === 'x-large' in my case locally), so this did not appear. But I'm not sure why.

Reply:
I know we don't copy _all_ production data to staging every where -- so maybe its that?